### PR TITLE
Responsive Design - Review

### PR DESCRIPTION
**Pull request by maelfosso**
Hi,

Please our project it's OK. Could you check it ?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,275 @@
...
+              <a href="#">
+                <div class="cover-logo"></div>
+                <div id="cover-time">Friday — March 15, 2019</div>
+                <header class="cover-header">
+                  <h5 class="cover-section">game</h5>
+                  <h2 class="cover-title">
+                    Who are you siding with in the Spotify-Apple antitrust war?
+                  </h2>
+                  <ul class="cover-header-bottom">
+                    <li> <a href="#">by Abhimanyu</a> </li>
```
Comment (inhaq) [inline index.html]: `a` shouldn't be inside `a` tag

**Review by inhaq — CHANGES_REQUESTED** [review]
### Good work 🏆
Site is responsive, that's great. Here are a few things to take care of:
 - Navbar shouldn't have `margin-bottom`.
 - Images aren't responsive. 
 - Footer is missing.

Kindly do these tweaks and resubmit for review. If you need assistance, feel free to DM me via slack.

File: index.html (hunk 1 above)
Comment (maelfosso) [reply index.html]: Thanks. We solved it.

Comment (maelfosso) [reply index.html]: Don't understand *images aren't responsive*

**Review by inhaq — CHANGES_REQUESTED** [review]
Amazing work on the footer, but still images aren't responsive e.g. [these](https://snag.gy/L7U5Jb.jpg) images are stretched. Navbar has some [margin in between](https://snag.gy/O6QTJu.jpg) and for medium size, Article is behind the navbar. Kindly fix these and resubmit for review. DM me on slack if you need more help.

**Review by inhaq — APPROVED** [review]
### 👍🏻Good job, guys
The design looks stunning. Go ahead and submit a project completion form. Good luck with the next projects.

**Pull request merged**